tailwind.config.js

```diff
@@ -53,7 +53,7 @@ module.exports = {
     ...typePlugins,
     ...glyphPlugins,
     require("@tailwindcss/forms")({ strategy: "class" }),
-    require("@tailwindcss/line-clamp"),
+    require('@tailwindcss/line-clamp'),
```

Comment:
**Suggestion** It looks like the rest of the file is using `"`. I think we should keep that consistent.

Reply:
Updated!